.changeset/mean-cherries-press.md

```diff
@@ -0,0 +1,7 @@
+---
+"@khanacademy/wonder-blocks-clickable": minor
+"@khanacademy/wonder-blocks-dropdown": minor
+"@khanacademy/wonder-blocks-core": minor
```

Comment:
Related to our [thread about versioning in the other PR](https://github.com/Khan/wonder-blocks/pull/2345#discussion_r1909588990), would this also be a breaking change if consumer projects were relying on the `event.which/event.keyCode` functionality?

Reply:
Great question. I think yes, and this could be released along with the other breaking changes. So it's good timing!

Reply:
Hmm, now I'm rethinking `major` on this and wondering if `minor` is more appropriate. The move to `event.key` is internal, and not part of the public API. Curious to hear your thoughts @beaesguerra!

Reply:
Looks like I marked the [similar event.key change](https://github.com/Khan/wonder-blocks/pull/2376/files#diff-d3528e7b0e4be61064f2f91918d8d6f3e76c3585340c38ad444a3b8b43014a83R1-R5) for the SelectOpener as a `patch` change 😅 I think it is fine that we don't mark it as `major` since it is internal. It would be good to double check these areas in current usage though just to be sure there wasn't functionality relying on that! 

And since this would be released with the other combobox changes, it'll be part of the new major version as you mentioned!